src/app/api/manager/sources/[ingest_name]/[source_name]/thumbnail/route.ts

```diff
@@ -35,6 +35,7 @@ export async function GET(
     }
     return new NextResponse(Buffer.from(base64Image, 'base64'));
   } catch (e) {
+    console.error('Error fetching thumbnail:', e);
```

Comment:
Do we need to console.error the error? Could this either be removed or replaced with Log()?

Reply:
I'm updating it to a Log() instead, feel like we need the error-message because the catch doesn't leave a clear message telling what goes wrong and where.